server.js

```diff
@@ -71,6 +71,15 @@ module.exports = function(options) {
     uri: process.env.APPLICATION_URI
   });
 
+  server.register(require("hapi-auth-bearer-token"), function(err) {});
```

Comment:
we don't need to handle any error there? there could be an internal error there right?